lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -0,0 +1,63 @@
+# IMPORTANT NOTES ABOUT THIS FILE
+#
+# a) cardano-node does not parse 'yaml' but only 'json'. We use yaml as a nicer/simpler format
+#    and convert it as json when generating the underlying configuration for the node.
+#
+# b) the `systemStart` is hard-coded here to please the parser but is replaced dynamically
+#    by the same code generating the final node configuration for integration.
+#
+# c) initialFunds are defined as a list of singleton, and not as a key-value map like in the
```

Comment:
Hm.🤔 We don't really control the order when we fill the shelley faucets. Maybe this might cause some failures @KtorZ .

Maybe we have to make sure to always keep funds for a sequential wallet in a single tx (but we may have as many as 200 addresses to fill for a single wallet).

Reply:
This will be a problem for big wallets only, which might explain why the scenario is currently failing. I thought initial funds in the shelley genesis were ignored though?

Reply:
> I thought initial funds in the shelley genesis were ignored though?

Yes. 

(I came to think about it when reading the outdated documentation comment. Now updated.)